raft/node.go

```diff
@@ -344,8 +356,16 @@ func (n *node) run(rn *RawNode) {
 			cs := r.applyConfChange(cc)
 			if _, ok := r.prs.Progress[r.id]; !ok {
 				// block incoming proposal when local node is
-				// removed
-				if cc.NodeID == r.id {
+				// removed.
+				all := append([]uint64(nil), cs.Nodes...)
```

Comment:
I guess this isn't performance-critical, but two appends feels expensive in comparison to just doing two trivial for loops.

Reply:
Done.